src/background.js

```diff
@@ -1,4 +1,3 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
The original purpose was to disable this `electron` error in the `.eslintrc.js`, and it solved, but I don't know how.

Reply:
OK, it is not solved, so I have different behavior in my workspace.

Reply:
Solved